app/config/FrontendAppConfig.scala

```diff
@@ -145,6 +146,15 @@ class FrontendAppConfigImpl @Inject() (configuration: Configuration, langs: Lang
   override val barsBusinessAssessUrl: String =
     s"$barsBaseUrl${configuration.get[String]("microservice.services.bank-account-reputation.businessAssess")}"
 
+  private val accessibilityLookupBaseUrl: String =
+    configuration.get[Service]("microservice.services.accessibility-statement").baseUrl
+
+  private val accessibilityLookupServiceBaseUrl: String =
```

Comment:
OBSERVATION: I've done this in two stages so I don't end up with a real long line that looks like this
`override val accessibilityReportUrl: String = 
s"$accessibilityLookupBaseUrl${configuration.get[String]("microservice.services.accessibility-statement.serviceBase")}${configuration.get[String]("accessibility-statement.service-path")}"`